sentences/fr/_common.yaml

```diff
@@ -528,13 +544,14 @@ expansion_rules:
   ventilateurs: "[les ](ventilateurs|brasseurs d'air)"
   fenetre: "(fenetre[s]|fenêtre[s]|baie[s]|velux|vélux|lucarne[s])"
   appareil: "(appareil|machine|équipement)[s]"
+  capteur: "(capteur|sonde|détecteur)[s]"
+  media: "(morceau|chanson|élément|podcast|film|vidéo|épisode)"
```

Comment:
Would you mind adding "média" ?
I feel if the response for the Next intent is "Média suivant" "média" should be on the list too be to able to say "Média suivant sur la TV"